docs-website/docs/pipeline-components/agents-1/agent.mdx

```diff
@@ -133,6 +133,77 @@ print(response["messages"])
 print("Calc Result:", response.get("calc_result"))
 ```
 
+### Agents as Tools
+
+You can wrap an `Agent` using [`ComponentTool`](../../tools/componenttool.mdx) to create multi-agent systems where specialized agents act as tools for a coordinator agent.
+
+When wrapping an `Agent` as a `ComponentTool`, use the `outputs_to_string` parameter with `{"source": "last_message"}` to extract only the agent's final response text, rather than the entire conversation history with tool calls to keep the coordinator agent's context clean and focused.
```

Comment:
I think we need to replace the word 'conversation history' here. Not sure about the write wording though.
```suggestion
When wrapping an `Agent` as a `ComponentTool`, use the `outputs_to_string` parameter with `{"source": "last_message"}` to extract only the agent's final response text, rather than the execution trace with tool calls to keep the coordinator agent's context clean and focused.
```